### ISSUE TITLE
A11y_SQLAzureDataStudio_SQLServerBigDataCluster_ConfigurationProfile_Screenreader: Unnecessary table markup found for Feature section

### ISSUE THREAD

**Issue by shweta255** [microsoft/azuredatastudio#9343]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

#A11yMAS;#A11yTCS;#SQL Azure Data Studio;#DesktopWeb;#Win10;#MAS1.3.1;#NVDA;#Benchmark;#A11y_SQLAzureDataStudio;#A11ySev2;

**Environment Details:**
Version: 1.16.0-insider
Commit: 506c6a5
Date: 2020-02-17T05:43:18.399Z
VS Code: 1.42.0
Electron: 7.1.11
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Windows_NT x64 10.0.18363
Additional Details:
MAS References: MAS1.3.1
NVDA Version: 2019.2

**Repro Steps:**
1. Launch Azure Data Studio.
2. Select the Connections icon (if not already selected) from left bar.
3. Click on the ellipses '...' and select "New Deployment".
4. Select "SQL Server Big Data Cluster" card under deployment options.
5. Select the version & checkbox.
6. Navigate till Selet button & hit enter.
7. Launch NVDA screenereader
8. Navigate to feature section of radiobutton content using down arrow key

**Actual:**
Unnecessary table markup found for Feature section

**Expected:**
Feature section should not have table markup as there is no column present.

**User Impact:**
Screenreader users will face difficulty as there is not column present in the table.

**Attachment link reference:**
[9340_A11y_SQLAzureDataStudio_SQLServerBigDataCluster_ConfigurationProfile_Screenreader.zip](https://github.com/microsoft/azuredatastudio/files/4274761/9340_A11y_SQLAzureDataStudio_SQLServerBigDataCluster_ConfigurationProfile_Screenreader.zip)

**Comment by alanrenmsft** [comment]
@shweta255  this was implemented as per your team's previous testing: https://github.com/microsoft/azuredatastudio/issues/7933

and it doesn't have columns

**Comment by BhanuPrakashGit** [comment]
@alanrenmsft Could you please the reopened the bug as it should be verified the Trusted Tested.
Please provide the latest environment details where we can verify this and take action accordingly.

GithubTags:#Regressed:06-02-23;

**Comment by BhanuPrakashGit** [comment]
As "SQL Server Big Data Cluster" feature has been removed
**GithubTags:** #Closed;

**Comment by mstechie** [comment]
GitHubTags:#SQLADS-Win32-Feb2020;
